### ISSUE TITLE
summary() should report number of NA values

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1586]
I've been working on #1582 (single flag for all data) and find myself always trying to use `summary()` to see the results after passing through `handleFlags()`. I see now that `summary()` really should report the number of NA values. I think a good place for that would be just to the right of the `Dim.` column.

Since I don't think any users are using the result of `summary()` in further computations (which can be done by saving the output and parsing it ... not a crazy thing to do, but not something I've heard of a user doing!), I will go ahead and do this, without waiting for input from other developers over this long weekend.

```
                              Min.   Mean   Max.   Dim. OriginalName
    pressure [db]             11.4   2182.5 4398.3 24   CTDPRS      
    temperature [°C, IPTS-68] 2.2233 7.3839 23.554 24   CTDTMP      
    salinity [PSS-78]         35.07  36.081 36.692 24   CTDSAL      
    salinityBottle [PSS-78]   35.07  36.076 36.702 24   SALNTY      
    oxygen [μmol/kg]          144.9  191.59 217.6  24   OXYGEN      
    silicate [μmol/kg]        1.71   6.73   16.78  24   SILCAT      
    nitrite [μmol/kg]         0      0.022  0.08   24   NITRIT      
    NO2+NO3 [μmol/kg]         0      9.3387 23.73  24   NO2+NO3     
    phosphate [μmol/kg]       0      0.5175 1.39   24   PHSPHT      

```

**Comment by dankelley** [comment]
I've done this in my not-yet-pushed code, and it looks like below. I like it.  (I do not plan on pushing this code until I am quite happy with the fix for #1582.)

I will leave this issue open until I push the code, perhaps today, perhaps tomorrow, perhaps Monday.

```
* Data

                              Min.   Mean   Max.   Dim. NAs OriginalName
    pressure [db]             11.4   2182.5 4398.3 24   0   "CTDPRS"    
    temperature [°C, IPTS-68] 2.2233 7.3839 23.554 24   0   "CTDTMP"    
    salinity [PSS-78]         34.883 35.315 36.692 24   0   "CTDSAL"    
    salinityBottle [PSS-78]   34.889 35.315 36.702 24   0   "SALNTY"    
    oxygen [μmol/kg]          144.9  246.91 282.1  24   0   "OXYGEN"    
    silicate [μmol/kg]        1.71   16.852 36.52  24   0   "SILCAT"    
    nitrite [μmol/kg]         0      0.022  0.08   24   19  "NITRIT"    
    NO2+NO3 [μmol/kg]         0      14.619 23.73  24   0   "NO2+NO3"   
    phosphate [μmol/kg]       0      0.8925 1.39   24   0   "PHSPHT"   
```

PS. a reader might now guess why I wanted this ... I hadn't realized that some of the Nitrite values were `NA` in the original object. This was causing some problems in my test code. From now on, I won't get surprised by such things.

**Comment by dankelley** [comment]
Oh, I did this a while back and forgot to close the issue.
